bdr_solrizer/solrdocbuilder.py

```diff
@@ -173,6 +173,11 @@ def original_object(self):
     def ancestors(self):
         return [self.original_object, self.parent_object]
 
+    def is_image_child(self):
+        objects = list(self.rels_ext.objects(predicate=model_ns.hasModel))
```

Comment:
I'm unclear how this indicates that the current storage object is an image child.

Reply:
Me too. I'll fix it. Thanks.

Reply:
updated